ios/RNCPagerViewManager.m

```diff
@@ -7,6 +7,9 @@ @implementation RNCPagerViewManager
 
 RCT_EXPORT_MODULE(RNCViewPager)
 
+RCT_EXPORT_VIEW_PROPERTY(page, NSInteger)
+RCT_EXPORT_VIEW_PROPERTY(animated, BOOL)
```

Comment:
I have a question about the `onPageSelected` behavior.  Should it behave in the same way as uncontrolled flow? 

Because in controlled flow, this callback is not necessary to implement?

Reply:
I think so. When setting the page with 'page' props, the user will still be able to call some functions at the end of the page change transition.